File: beecrowd/1/1018/1018.cpp
#include <iostream>

int main() {
    int dinheiro, quantidade;
    std::cin >> dinheiro;

    int notas[7] = {100, 50, 20, 10, 5, 2, 1};
    std::cout << dinheiro << std::endl;
    for (int i = 0; i < 7; i++) {
        quantidade = dinheiro / notas[i];
        dinheiro -= quantidade * notas[i];
        std::cout << quantidade << " nota(s) de R$ " << notas[i] << ",00" << std::endl;
    }
    return 0;
}

File: beecrowd/1/1018/1018.cs
using System;

class URI {

    static void Main(string[] args) {
        int dinheiro, quantidade;
        dinheiro = int.Parse(Console.ReadLine());

        int[] notas = {100, 50, 20, 10, 5, 2, 1};
        Console.WriteLine("{0}", dinheiro);
        for (int i = 0; i < 7; i++) {
            quantidade = dinheiro / notas[i];
            dinheiro -= quantidade * notas[i];
            Console.WriteLine("{0} nota(s) de R$ {1},00", quantidade, notas[i]);
        }
    }
}

